### PR TITLE
Fix billing query

### PR DESCRIPTION
**Pull request by zugao**
* Fix appcat:metering query for the cronjobs
* Increasesed quotas for objects according to https://kb.vshn.ch/appuio-cloud/how-to/day2ops/override-default-quota.html
* Decreased the amount of failed and successful jobs to be retained.

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

**Comment by wejdross** [comment]
By default was it 59 hours with one hour step?

**Comment by zugao** [comment]
> By default was it 59 hours with one hour step?

minutes, everything should be in minutes, and yes it's 59 minutes due to some issues with promql when setting 60m

File: component/billing.jsonnet [hunk 1]
```diff
@@ -157,8 +158,8 @@ local generateCloudAndManaged = function(name)
...
 
-  local managedQuery = 'sum_over_time(appcat:metering{label_appuio_io_billing_name="appcat-' + queryName + '",label_appcat_vshn_io_sla="%s", tenant_name!="APPUiO"}[59:1])/60';
-  local cloudQuery = 'sum_over_time(appcat:metering{label_appuio_io_billing_name="appcat-' + queryName + '",label_appcat_vshn_io_sla="%s", tenant_name="APPUiO"} * on(label_appuio_io_organization) group_left(sales_order) label_replace(appuio_control_organization_info{namespace="appuio-control-api-production"}, "label_appuio_io_organization", "$1", "organization", "(.*)")[59:1])/60';
+  local managedQuery = 'sum_over_time(appcat:metering{label_appuio_io_billing_name="appcat-' + queryName + '",label_appcat_vshn_io_sla="%s", tenant_name!="APPUiO"}[59m:1m])/60';
+  local cloudQuery = 'sum_over_time(appcat:metering{label_appuio_io_billing_name="appcat-' + queryName + '",label_appcat_vshn_io_sla="%s", tenant_name="APPUiO"} * on(label_appuio_io_organization) group_left(sales_order) label_replace(appuio_control_organization_info{namespace="appuio-control-api-production"}, "label_appuio_io_organization", "$1", "organization", "(.*)")[59m:1m])/60';
```
Comment (wejdross) [inline component/billing.jsonnet:162]: so if both queires are identical, why did You add `m`? `59` -> `59m`

Comment (zugao) [reply component/billing.jsonnet:162]: if I don't set m it's not a valid promql anymore, it fails

Comment (wejdross) [reply component/billing.jsonnet:162]: ok, resolved

**Review by wejdross — APPROVED** [review]
can't see how would that be harmful, LGTM

**Review by Kidswiss — CHANGES_REQUESTED** [review]

File: component/main.jsonnet [hunk 2]
```diff
@@ -85,7 +86,11 @@ local ns = kube.Namespace(params.namespace) {
     labels+: {
       'openshift.io/cluster-monitoring': 'true',
     } + params.namespaceLabels,
-    annotations+: params.namespaceAnnotations,
+    annotations+:
+      if !appuioManaged then {
+        'resourcequota.appuio.io/organization-objects.jobs': '50',
```
[the remark this answers is not in the document]
Comment (zugao) [reply component/main.jsonnet:91]: for all objects no?

[the remark this answers is not in the document]
Comment (zugao) [reply component/main.jsonnet:91]: increased ro 300 :)

**Review by Kidswiss — APPROVED** [review]

**Comment by Kidswiss** [comment]
Once we verified that the metrics are correct, we need to backfill the weekend as well.
